code/Controllers/ContentController.php

```diff
@@ -419,17 +411,17 @@ public function getViewer($action)
         }
 
         // Find templates for the controller + action together - e.g. PageController_action.ss
-        $templatesFound[] = SSViewer::get_templates_by_class(static::class, $action, Controller::class);
+        $templatesFound[] = SSViewer::get_templates_by_class(static::class, $action ?? '', Controller::class);
```

Comment:
`$action` isn't typed, and `null` is pretty common to say "there's no action" so just a bit of type safety here.